components/form/demo/global-state.md

```diff
@@ -58,9 +58,9 @@ class Demo extends React.Component {
     },
   };
   handleFormChange = (changedFields) => {
-    this.setState({
-      fields: { ...this.state.fields, ...changedFields },
-    });
+    this.setState(({ fields }) => ({
+        fields: { ...fields, ...changedFields }
```

Comment:
Keep two spaces as indent.

Reply:
Do I need to make another commit and push it on my fork? (sorry for the silly question, it's my first PR)

Reply:
No, you can fix and push it in this PR.